data/hai/hai culture conversations.txt

```diff
@@ -329,3 +329,55 @@ mission "Hai Film Release"
 				decline
 			label translator
 			`	You find a movie theater showing Likeet's film, but your translator is of little help here. The movie is sparse on dialog, and the few lines spoken in the movie seem to be cryptic references to Hai literature and philosophy. The most you are able to make out is that the movie is about an unseen monster inhabiting the station that slowly drives those it meets insane. It seems to be a hit with the Hai audience, though, and you leave the theater feeling distinctly unnerved.`
+
+
+
+mission "Sick of Acorn Bread"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		credits > 50
+		random < 1
+	on offer
+		conversation
+			`With some time to kill while your ship is being serviced by the Hai dockhands, you enter the spaceport searching for a bite to eat. Restaurants advertise their menus in bright, attention-grabbing colors on their windows; amongst the selection, you spot one that promises "authentic Hai cuisine," along with some large text printed in the Hai language which you don't understand. Next to it, you see a human street food vendor offering snacks and ready-made meals.`
```

Comment:
The translator you get later includes text to speech, so you'll either want to have a branch here on the text if you've got the translator, or remove the specific statement that you don't understand the language.